### ISSUE TITLE
Dead 💀,  alive 🕺, or try and bring 👩‍⚕️back to life 🧟‍♂️

### ISSUE THREAD

**Issue by ipatch** [caskformula/homebrew-caskformula#75]
Not sure what the current state of this repo is, but I ended spending some time today to build Inkscape on macOS 10.13.6 from the master branch as of February 1, 2019.

That said, ...looked at the people associated with is repo, and saw no people associated with **caskformula**.

- haven't seen a commit pushed in several months
- several issues need handling
- several PR's need to be triaged, etc etc.

So before I even waste my time to submit a PR, I'll probably just setup a Inkscape formula in a homebrew tap I setup a while back.  That said, I managed to get Inkscape to compile this evening, and apparently launching the **X11.app** is no longer required, and HiDPI appears to be a thing now.  Happy new years 🍾

<img width="1217" alt="tech-inkscape-macos-hidpi" src="https://user-images.githubusercontent.com/613805/52159838-66d57000-266f-11e9-9917-013c162257bd.png">

**Comment by lljr** [comment]
@ipatch is the homebrew formula ready? I'm having serious issues with this one...

**Comment by ipatch** [comment]
@avocadoras 

Not yet, working on a laundry list of items at the moment, and not sure if I'll get around to posting a formula today, as I don't want to post something incomplete, and ends up making people more frustrated.  A couple of things I'm working on,

- figure out how to build an **.app** image / bundle, so I just have a **Inksape.app** as it presently stands, I have to invoke inkscape from a CLI to get it to open. 👎
- figure out how to build a **.dmg**, so I can have a homebrew cask formula that will make the installation process a whole lot easier, because building inkscape from the gitlab source is quite a time consuming task.
- figure out how to get "floating windows", the build I produced last night, opens all windows / dialogs in a kind of tab like fashion, which is not ideal for dialogs such as a file open and what not.

So there definitely is a laundry list of things that need to get done before I'd feel comfortable publishing a formula.  That said, 🤞, hopefully I'll get a formula setup that builds inkscape from source if one so chooses, or maybe just post the steps to compile from source and then make a rather simple formula that installs inkscape from a dmg as the current cask formula does when running

```shell
brew cask install inkscape
```

**Comment by lljr** [comment]
@ipatch is there a way you could just distribute a binary image so as to use is as a dependency to other processes that need it like `pdflatex`? I don't actually use nor need the inkscape GUI... could be a good option for those that really need the package asap...

**Comment by ipatch** [comment]
@avocadoras correct me if I'm wrong but there is a DMG that has already been generated a while back that contains **Inkscape.app**

[Downloadable DMG](https://inkscape.org/release/inkscape-0.92.2/mac-os-x/107/dmg/dl/)

```
https://inkscape.org/release/inkscape-0.92.2/mac-os-x/107/dmg/dl/
```

**Comment by lljr** [comment]
@ipatch yeah had to use that. But will be patiently waiting for your formula. wish I knew more so I could actually help you

**Comment by dehesselle** [comment]
@ipatch As you probably know, the Inkscape devs are looking for someone to help out with building and packaging Inkscape for macOS. I want to give it a try, and the first step is taking a look at what is already there. Since I knew that Inkscape is in homebrew, I came here and found this issue.

I'd like to avoid to reinvent the wheel, so if you are willing to share what you've done so far - no matter how unfinished/unpolished you think it is - that would be a great timesaver (or at least a way to compare notes).

**Comment by ipatch** [comment]
@dehesselle 

[notes](https://github.com/ipatch/homebrew-us-05/blob/master/inkscape/inkscape-building-for-macOS.md)

☝️ those are the notes I recorded while messing around with Inkscape & tooling on macOS.

**Comment by dehesselle** [comment]
Thank you. I've setup a repo for my notes and link it back [here](https://github.com/dehesselle/mibap) so you can see that it's not a one-way street.

_edit: add link_
